appstudio-utils/util-scripts/lib/fetch/rekor.sh

```diff
@@ -16,21 +16,15 @@ rekor-uuid-entry() {
 }
 
 # Extract the log index from a transparency url
-# Todo: Make it more robust, use perl maybe
 log-index-from-url() {
   local url=$1
-
-  # Assume it ends with something like '?logIndex=1234'
```

Comment:
Might want to keep this comment but update it, e.g "Assume it has a url param called logIndex"